### PR TITLE
Data tab tests

### PR DESCRIPTION
**Pull request by naterush**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context, and the specification if there is one. 

# Testing

Please provide a list of the ways you can "access" or use the functionality. Please try and be exhaustive here, and make sure that you test everything you list.

- [ ] I have tested this on real data that is reasonable and large
- [ ] If I changed the interaction with JupyterLab, I tested that it does not break other programs (like VS Code), and tested that it works "multiple times" in the same notebook.

# Documentation

Note if any new documentation needs to addressed or reviewed.

### CODE REVIEW

File: tests/streamlit_ui_tests/streamlit.spec.ts [hunk 1]
```diff
@@ -320,6 +336,359 @@ test.describe('Home Tab Buttons', () => {
...
+  });
+
+  test('Test Anti-merge', async ({ page }) => {
+    const mito = await getMitoFrameWithTestCSV(page);
+    await importCSV(page, mito, 'test.csv');
+    await clickTab(page, mito, 'Insert');
+
+    await clickButtonAndAwaitResponse(page, mito, 'Anti-merge');
+
+    await expect(mito.getByText('Merge Dataframes')).toBeVisible();
```
Comment (marthacryan) [inline tests/streamlit_ui_tests/streamlit.spec.ts:414]: Shouldn't this be `checkOpenTaskpane`?

**Review by marthacryan — COMMENTED** [review]
These are looking awesome!! I had some small comments but overall I love the structure of the tests and usage of utils.

File: tests/streamlit_ui_tests/streamlit.spec.ts [hunk 2]
```diff
@@ -8,10 +8,22 @@ const getMitoFrame = async (page: Page): Promise<FrameLocator> => {
 };
 
 const importCSV = async (page: Page, mito: FrameLocator, filename: string): Promise<void> => {
+  // Case if there are 0 .tab 
+  let tabCount = 0;
+  try {
```
[the remark this answers is not in the document]
Comment (naterush) [reply tests/streamlit_ui_tests/streamlit.spec.ts:13]: Will update comment. The locator throws an error if there are 0

**Comment by naterush** [comment]
LGTM when tests pass

**Pull request merged**